src/types/todo.ts

```diff
@@ -0,0 +1,12 @@
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user?: {
+    id: number;
+    name: string;
+    username: string;
+    email: string;
+  } | null;
+}
```

Comment:
```suggestion
  user?: User | null;
}
```

Reply:
let's fix that
<img width="626" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/01796e2c-e823-49da-9653-4b60718b8524">